test/integration/regression/test_regression_bug335.py

```diff
@@ -0,0 +1,85 @@
+import sys
+from pathlib import Path
+
+import pytest
+from sqlalchemy import (
+    Column,
+    Integer,
+    MetaData,
+    String,
+    Table,
+    create_engine,
+    insert,
+)
+
+PROJECT_ROOT = Path(__file__).parent / ".." / ".." / ".."
+SCRIPTS = PROJECT_ROOT / "scripts"
+ODBC_DRIVER = PROJECT_ROOT / "driver" / "libexaodbc-uo2214lv2.so"
+sys.path.append(f"{SCRIPTS.resolve()}")
+
+
+@pytest.fixture
+def pyodbc_connection_string(exasol_config):
+    config = exasol_config
+    return (
+        f"exa+pyodbc://{config.username}:{config.password}@{config.host}:{config.port}/"
+        f"?DEFAULTPARAMSIZE=200&INTTYPESINRESULTSIFPOSSIBLE=y"
+        "&FINGERPRINT=NOCERTCHECK&CONNECTIONLCALL=en_US.UTF-8&driver=EXAODBC"
+    )
+
+
+@pytest.fixture()
+def test_schema(control_connection):
```

Comment:
What does the fixture control_connection mean here, in the implementation it looks like a connection to the test db?

Reply:
Do you mean with control, system under test or check or something like that?